integration-test/helpers/mocks.webkit.js

```diff
@@ -128,7 +128,6 @@ export const macosWithoutOverlay = () => {
             'emailHandlerGetAddresses',
             'emailHandlerCheckAppSignedInStatus',
             'pmHandlerGetAutofillInitData',
-            'pmHandlerStoreData',
```

Comment:
We renamed this `storeFormData` to align with other platforms, and it was already in the list.